spec/controllers/api/v1/children_controller_spec.rb

```diff
@@ -7,7 +7,7 @@
   context 'on GET to index' do
     context 'requesting json representation', api_specification: true do
       let(:children_schema) do
-        schema_for('ontology/commands/children')
+        schema_for_command('ontology/children', :get, 200)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.